source/_integrations/shelly.markdown

```diff
@@ -122,4 +122,5 @@ Not all devices support all input events. You can check on [Shelly API Reference
 ## Known issues and limitations
 
 - Only supports firmware 1.8 and later
+- Shelly4Pro is not supported (device based on old CoAP v1 protocol)
```

Comment:
I think we're using `Shelly 4Pro` name, with a space, in `aioshelly`.

Reply:
Done